src/krfctl/freebsd/freebsd.c

```diff
@@ -0,0 +1,99 @@
+#include <stdio.h>
+#include <string.h>
+#include <sys/errno.h>
+#include <errno.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+#include <stdlib.h>
+#include <err.h>
+#include <string.h>
+#include <sys/sysctl.h>
+
+#include "../krfctl.h"
+
+/* control will interpret any number larger than its syscall table
+ * as a command to clear all current masks.
+ * it's a good bet that linux will never have 65535 syscalls.
+ */
+#define CLEAR_MAGIC 65535
```

Comment:
Let's dedupe this from here and `linux.c` and put it in `krfctl.h`.

Reply:
Also `s/linux/our supported platforms/`

Reply:
There a slight difference, which is that FreeBSD needs `CLEAR_MAGIC` to be an integer, and Linux uses it as a string, so I thought it was worth having them seperate

Reply:
Ah, didn't notice. Yeah, that's fine for now then.